app/models/allocation.rb

```diff
@@ -68,11 +68,19 @@ def case_worker
     # deallocation will have a nil case worker id
   end
 
-  def allocating?
-    @allocating
+  private
+
+  def should_reallocate_claims
```

Comment:
Just a naming suggestion: personally like the boolean `?` thing
```
can_allocate_claims?
```

Reply:
Good call... 
> naming things is hard
:(